### PR TITLE
Make user Read and Delete for Admin

### PR DESCRIPTION
**Pull request by Tomoki108**
***Commands to run***
-none

***Preconditions***
-Login with any account.
-Click "User list"' nav link and go to user list page.
-Click delete link to test delete function.

***Expected outputs***
-Admin (for now all users, since authentication function is not yet made.) can see all users in user list.
-Admin (for now all users, since authentication function is not yet made.) can delete user account.

***asana***
-https://app.asana.com/0/984197702640917/984594424660437

### CODE REVIEW

File: app/Http/Controllers/CategoryController.php [hunk 1]
```diff
@@ -9,8 +9,7 @@ class CategoryController extends Controller
 {
     public function index()
     {
-        $category = new Category;
-        $categories = $category->getCategories(5);
+        $categories = Category::orderBy('id', 'DESC')->paginate(5);
```
Comment (jeremiahC) [inline app/Http/Controllers/CategoryController.php]: alway remember the pattern of MVC, orderBy is model related function, make a function in model category that will do this behavior and the controller will just call that function.

File: routes/web.php [hunk 2]
```diff
@@ -41,3 +41,6 @@
 Route::post('/admin/word/update', 'WordController@update');
 Route::get('/admin/word/delete/{word}', 'WordController@delete');
 
+//User RD functions for Admin
+Route::get('/admin/user', 'UserController@index');
+Route::get('/admin/user/delete/{user}', 'UserController@delete');
```
Comment (jeremiahC) [inline routes/web.php]: delete method should not be get request, it must be delete request.

note: remember the HTTP verbs which are GET, POST, PUT, PATCH and DELETE

Comment (Tomoki108) [reply routes/web.php]: Okay, I understand. And It's not only about this route. I'll fix all delete route.

File: app/Http/Controllers/UserController.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+{
+    public function index()
+    {
+        return view('admin.userList')->with('users',  User::orderBy('id', 'DESC')->paginate(20));
+    }
+
+    public function delete(User $user)
+    {
+        $user->delete();
+        return redirect()->action('UserController@index', ['users' => User::orderBy('id', 'DESC')->paginate(20)])->with('Success', 'The user is deleted successfully.');
```
Comment (jeremiahC) [inline app/Http/Controllers/UserController.php]: i think no need to declare again user data if you will just redirect to index page, index method already do the job of displaying the list users.

**Review by jeremiahC — APPROVED** [review]

**Pull request merged**